cuchem/cuchem/benchmark/megamolbart.py

```diff
@@ -16,23 +16,35 @@
 
 from cuchem.wf.generative import MegatronMolBART
 from cuchem.wf.generative import Cddd
-from cuchem.datasets.loaders import ZINC15_TestSplit_20K_Samples, ZINC15_TestSplit_20K_Fingerprints
-from cuchem.metrics.model import Validity, Unique, Novelty, NearestNeighborCorrelation, Modelability
-
-gene_list = ["ABL1", "ACHE", "ADAM17", "ADORA2A", "ADORA2B", "ADORA3", "ADRA1A", "ADRA1D", 
-             "ADRB1", "ADRB2", "ADRB3", "AKT1", "AKT2", "ALK", "ALOX5", "AR", "AURKA", 
-             "AURKB", "BACE1", "CA1", "CA12", "CA2", "CA9", "CASP1", "CCKBR", "CCR2", 
-             "CCR5", "CDK1", "CDK2", "CHEK1", "CHRM1", "CHRM2", "CHRM3", "CHRNA7", "CLK4", 
-             "CNR1", "CNR2", "CRHR1", "CSF1R", "CTSK", "CTSS", "CYP19A1", "DHFR", "DPP4", 
-             "DRD1", "DRD3", "DRD4", "DYRK1A", "EDNRA", "EGFR", "EPHX2", "ERBB2", "ESR1", 
-             "ESR2", "F10", "F2", "FAAH", "FGFR1", "FLT1", "FLT3", "GHSR", "GNRHR", "GRM5", 
-             "GSK3A", "GSK3B", "HDAC1", "HPGD", "HRH3", "HSD11B1", "HSP90AA1", "HTR2A", 
-             "HTR2C", "HTR6", "HTR7", "IGF1R", "INSR", "ITK", "JAK2", "JAK3", "KCNH2", 
-             "KDR", "KIT", "LCK", "MAOB", "MAPK14", "MAPK8", "MAPK9", "MAPKAPK2", "MC4R", 
-             "MCHR1", "MET", "MMP1", "MMP13", "MMP2", "MMP3", "MMP9", "MTOR", "NPY5R", 
-             "NR3C1", "NTRK1", "OPRD1", "OPRK1", "OPRL1", "OPRM1", "P2RX7", "PARP1", "PDE5A", 
-             "PDGFRB", "PGR", "PIK3CA", "PIM1", "PIM2", "PLK1", "PPARA", "PPARD", "PPARG", 
-             "PRKACA", "PRKCD", "PTGDR2", "PTGS2", "PTPN1", "REN", "ROCK1", "ROCK2", "S1PR1", 
+from cuchem.benchmark.data import PhysChemEmbeddingData, SampleCacheData, ZINC15TrainDataset
+from cuchem.benchmark.datasets.fingerprints import ZINC15TestSplitFingerprints
+from cuchem.benchmark.datasets.molecules import ZINC15TestSplit
+from cuchem.benchmark.metrics.sampling import Validity, Unique, Novelty
+from cuchem.benchmark.metrics.embeddings import NearestNeighborCorrelation, Modelability
+from cuchem.benchmark.datasets.molecules import (ChEMBLApprovedDrugsPhyschem,
+                                                 MoleculeNetESOLPhyschem,
+                                                 MoleculeNetFreeSolvPhyschem,
+                                                 MoleculeNetLipophilicityPhyschem )
+from cuchem.benchmark.datasets.fingerprints import (ChEMBLApprovedDrugsFingerprints,
+                                                    MoleculeNetESOLFingerprints,
+                                                    MoleculeNetFreeSolvFingerprints,
+                                                    MoleculeNetLipophilicityFingerprints )
+from cuchem.benchmark.datasets.bioactivity import (ExCAPEBioactivity, ExCAPEFingerprints)
+
+gene_list = ["ABL1", "ACHE", "ADAM17", "ADORA2A", "ADORA2B", "ADORA3", "ADRA1A", "ADRA1D",
```

Comment:
The ExCAPE code can be removed I believe -- failure to do so was an oversight on my part. This can wait until later too, since we will need to consolidate all the benchmark scripts.

Reply:
Done